tools/node_modules/eslint/node_modules/acorn-jsx/package.json

```diff
@@ -21,7 +21,7 @@
   "_resolved": "https://registry.npmjs.org/acorn-jsx/-/acorn-jsx-4.1.1.tgz",
   "_shasum": "e8e41e48ea2fe0c896740610ab6a4ffd8add225e",
   "_spec": "acorn-jsx@^4.1.1",
-  "_where": "/Users/cjihrig/iojs/node/tools/eslint-tmp/node_modules/eslint/node_modules/espree",
+  "_where": "/Users/about_hiroppy/Programming/nodejs/node/tools/eslint-tmp/node_modules/eslint/node_modules/espree",
```

Comment:
It appears that most of the 179 `package.json` file changes (of the 205 total changed files here) are simply changes like this. This is definitely churn we don't need. Any chance we can omit the package.json files that only change the `_where` path from the commit? I'll work on a PR to update `update-eslint.sh` to do this automatically.

Reply:
(PR: https://github.com/nodejs/node/pull/21819)

Reply:
good catch! thanks @Trott